upi/openstack/01_security-groups.yaml

```diff
@@ -149,6 +149,15 @@
       port_range_min: 2379
       port_range_max: 2380
 
+  - name: 'Create master-sg rule "etcd" for pod network'
```

Comment:
This task and the one in line 204 could be added in a different file and just include that file here based on:
"when: os_networking_type == "CiscoACI"

for example something like 

- name: Configure Cisco ACI  security groups: 
  includde : cisco_aci_security_groups.yaml
  when: os_networking_type == "CIscoACI" 

Then the cisco_aci_security_groups.yaml looks like:

---
- tasks:
      -name: 'Create master-sg rule "etcd" for pod network'
       os_security_group_rule:
      security_group: "{{ os_sg_master }}"
      protocol: tcp
      remote_ip_prefix: "{{ cluster_network_cidrs }}"
      port_range_min: 2379
      port_range_max: 2380

      - name: 'Create worker-sg rule "DNS (UDP)"'
    os_security_group_rule:
      security_group: "{{ os_sg_worker }}"
      remote_ip_prefix: "{{ cluster_network_cidrs }}"
      protocol: udp
      port_range_min: 53
      port_range_max: 53

... and any other ports that need to be added.